src/styles/blocks/header.scss

```diff
@@ -0,0 +1,76 @@
+.header {
+  background-color: $background-color-special-white;
```

Comment:
The variable `$background-color-special-white` is used but not defined. Please define this variable to ensure the background color is applied correctly.